templates/step/viewimg.html.ep

```diff
@@ -1,23 +1,43 @@
 <div class="aligncenter" data-image="<%= url_for('test_img', filename => $screenshot) %>" id="step_view">
-    Candidate needle:
```

Comment:
Adding this to viewimg is IMO not a bright idea. You want the same for wait_serial as well.